src/main/java/com/bankservice/bankservice/repository/RoleRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.bankservice.bankservice.repository;
+
+import com.bankservice.bankservice.model.Role;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+    @Query("FROM roles r WHERE r.roleName =:roleName")
+    Optional<Role> getByRoleName(@Param("roleName") String roleName);
```

Comment:
```suggestion
    Optional<Role> getByRoleName(Role.RoleName name);
```
Spring Data JPA is capable of automatically build the query in many cases based on the method name and parameters)